### DISCUSSION TITLE
Concern about privacy

### DISCUSSION THREAD

**Discussion by Gravity2106** [Extended-Thunder/send-later#413] [Q&A]
Hello,

Why does the download page say "This plugin can: show you notifications" but when you download it it asks you to give full control? Why does it need it? And why does the information between the download page and the "reality" not match?

**Comment by jph-sendlater** [comment]
You'd have to ask the addons.thunderbird.net maintainers why the download page is incorrect. As for the other question, that has been answered **many** times already. See this thread, which is linked from the FAQ: https://github.com/Extended-Thunder/send-later/discussions/250

**Reply by Gravity2106** [reply]
Thank you. I had looked for "privacy" but it did showed anything useful.
